Source/ASTableView.mm

```diff
@@ -1695,6 +1695,11 @@ - (NSUInteger)numberOfSectionsInDataController:(ASDataController *)dataControlle
   }
 }
 
+- (CGRect)dataController:(ASDataController *)dataController rectForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+  return [self rectForRowAtIndexPath:indexPath];
```

Comment:
We should be careful to subtract the separator height here if needed (look at `tableView:heightForRowAtIndexPath:` for info).